GitUI/GitUICommands.cs

```diff
@@ -523,29 +523,19 @@ public bool StartCheckoutBranch(IWin32Window owner, string branch, bool remote,
             });
         }
 
-        public bool StartCheckoutBranch(IWin32Window owner, string branch, bool remote)
-        {
-            return StartCheckoutBranch(owner, branch, remote, null);
-        }
-
         public bool StartCheckoutBranch(IWin32Window owner, string[] containRevisons)
         {
             return StartCheckoutBranch(owner, "", false, containRevisons);
         }
 
-        public bool StartCheckoutBranch(IWin32Window owner)
-        {
-            return StartCheckoutBranch(owner, "", false, null);
-        }
-
         public bool StartCheckoutBranch(string branch, bool remote)
         {
-            return StartCheckoutBranch(null, branch, remote, null);
+            return StartCheckoutBranch(null, branch, remote);
         }
 
         public bool StartCheckoutBranch()
         {
-            return StartCheckoutBranch(null, "", false, null);
+            return StartCheckoutBranch(null);
         }
 
         public bool StartCheckoutRemoteBranch(IWin32Window owner, string branch)
```

Comment:
📝 This appears to be another overload of `StartCheckoutBranch` with a different name.

Reply:
Yep, it could be inlined into the caller. There are several examples like this, even with overloads of the same name. Would be interesting to see what was left after such inlines.